src/app/public/theming/fixtures/theme-if-test.component.ts

```diff
@@ -14,14 +17,27 @@ import {
   SkyThemeSettings
 } from '../theme-settings';
 
+import {
+  SkyThemeIfDirective
+} from '../theme-if.directive';
+
 @Component({
   selector: 'app-theme-if-test',
   templateUrl: './theme-if-test.component.html'
 })
-export class SkyThemeIfTestComponent {
+export class SkyThemeIfTestComponent implements OnInit {
   public themeSettings: SkyThemeSettings;
+  @ViewChildren(SkyThemeIfDirective) private testDirectives!: QueryList<SkyThemeIfDirective>;
+
+  public doCheck(): void {
+    if (this.testDirectives?.length) {
+      this.testDirectives.forEach(testDirective => {
+        testDirective.ngOnChanges();
+      });
+    }
+  }
 
-  constructor() {
+  public ngOnInit(): void {
```

Comment:
Lifecycle hooks should be added before any other public methods. (We need to add this to the style guide.)

Reply:
TSLint wants the constructor first.